src/cljs/athens/types/tasks/events.cljs

```diff
@@ -0,0 +1,51 @@
+(ns athens.types.tasks.events
+  "`re-frame` events for `[[athens/task]]` entity type."
+  (:require
+    [athens.common-db                     :as common-db]
+    [athens.common-events                 :as common-events]
+    [athens.common-events.graph.composite :as composite]
+    [athens.common-events.graph.ops       :as graph-ops]
+    [athens.common.logging                :as log]
+    [athens.db                            :as db]
+    [re-frame.core                        :as rf]))
+
+
+(defn- save-prop-value
```

Comment:
This exact functionality already exists as the `:properties/update-in` event. For instance, the cover image functionality uses it like this:
```clj
(fn [url]
  (dispatch [:properties/update-in [:node/title title] [":header/url"] 
             (fn [db uid] [(graph-ops/build-block-save-op db uid url)])]))
```